apps/sso/components/login-card/login-card.tsx

```diff
@@ -105,8 +105,8 @@ export function LoginCard() {
 export default LoginCard;
 
 const StyledLoginCard = styled.div`
```

Comment:
nit: Similar to the Input components, the Card components all have the same CSS I believe, so you could extract them into one single component to prevent having multiple copies of code